.github/workflows/test-suite.yml

```diff
@@ -32,3 +37,4 @@ jobs:
         run: "scripts/test"
       - name: "Enforce coverage"
         run: "scripts/coverage"
+        if: "matrix.os == 'ubuntu'"
```

Comment:
We probably don't need this conditional once we've fixed up all the tests to pass okay on windows, right?

Reply:
scritps/coverage only works on unix